Rakefile

```diff
@@ -0,0 +1,6 @@
+require "bundler/gem_tasks"
```

Comment:
Rake `Ruby mAKE` is just a Ruby DSL for defining interdependent tasks, ostensibly for manipulating files but it is used generally for any isolated task that may or may not have dependencies on other tasks and needs to be run from the command line.

Here's a brief example of one: https://github.com/rocketmade/rock_doc/blob/master/lib/tasks/rock_doc_tasks.rake